front-end-notes/ajax.md

```diff
@@ -0,0 +1,25 @@
+## What is AJAX?
+
+#### AJAX is an acronym standing for Asynchronous JavaScript and XML. The technology helps us to load data from the server without a browser page refresh. This technology is used is popular sites such as Facebook.
+
+

```

Comment:
I would LOVE to see and example of an ajax call in code, and instructions if there's something I need to install in my project.